### ISSUE TITLE
[Feature] Add x-scheme-handler/* branch to account

### ISSUE THREAD

**Issue by PRESFIL** [sandsmark/selectdefaultapplication#2]
`x-scheme-handler/*` branch is used to set the default browser for the `*` protocol, but it cannot be edited from `selectdefaultapplication`. It would be nice to have such an opportunity.

It might even be better to display all the branches and types encountered.

**Comment by magnus-ISU** [comment]
Interesting. This is a trivial change, though it seems that `Qt` reports anything with `x-scheme-handler` as invalid even if it seems like it should be fine. Running my fork with the `-V` option we get tons of
```
In file  "Signal"  mimetype  "x-scheme-handler/sgnl"  is invalid. Ignoring...
```
I could add in a hardcoded exception for `x-scheme-handler/*` to be considered valid overriding what `Qt` claims. Guess I'll make a different pull request for that which can be accepted after my current changes.

**Comment by magnus-ISU** [comment]
Hmm maybe won't be as easy as I hoped

![image](https://user-images.githubusercontent.com/75949275/147772270-79adb5fa-b135-4e54-bfab-4e10e26d0e56.png)

There's no icon file associated with these, and there also seems to be some bugs regarding re-parsing them into `QMimeTypes` and back

**Comment by PRESFIL** [comment]
I said `*`, actually the question was specifically about `x-scheme-handler/http` and
`x-scheme-handler/http`. I just thought that it would be nice to implement this whole
tree since it is missing.

> Running my fork with the -V option we get tons of
> ```In file  "Signal"  mimetype  "x-scheme-handler/sgnl"  is invalid. Ignoring..```

What about `x-scheme-handler/http`? `xdg-mime` works with it correctly:

```sh
xdg-mime query default x-scheme-handler/https
firefox.desktop

grep 'x-scheme-handler/http' /usr/share/applications/mimeinfo.cache
```

**Comment by magnus-ISU** [comment]
I've figured out all the problems and the feature is now implemented in my fork. Just got to wait for it to get upstreamed

**Comment by PRESFIL** [comment]
Oh thank you! Looking at the @sandsmark activity since November, it won't be soon.

**Comment by magnus-ISU** [comment]
Yeah lol, but you can compile and install it yourself without too much trouble. If you're on arch let me know and I can send you a PKGBUILD, but otherwise it shouldn't be too difficult to package for yourself (or even easier to just drop in `/usr/local/bin`)

**Comment by PRESFIL** [comment]
Yes, I'm on an Arch-based distribution.

**Comment by magnus-ISU** [comment]
https://gist.github.com/magnus-ISU/dc023148df8b5ae045c0f37a934ded34#file-pkgbuild-L2

**Comment by PRESFIL** [comment]
All right, thank you. You have seriously reworked the layout... *"Small changes"*

**Comment by magnus-ISU** [comment]
=D

**Comment by magnus-ISU** [comment]
If there's anything you think should be different, let me know. The only problem the new layout might introduce I think is that it doesn't behave well if the window size is small (is there a good way to make a responsive layout in Qt?). But the added information makes it worth it I think. Do you disagree with any decisions I made?

EDIT: also I pushed a small change a moment ago that makes it so that it behaves better when it has less space to work with, so rerun `makepkg -fsi` to try it out if that is your concern

EDIT: and again more big changes, this time fixing one of the major differences which I had accidentally removed each mimetype's description
